tests/test_json_validation.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+import jsonschema
+from jsonschema import validate
+from app.services.g6importService import validate_json
+
+
+def test_all_schemas_are_valid():
+    for file_name in os.listdir('schemata'):
+        if os.path.isfile('schemata/%s' % file_name):
+            print('Testing schema: %s' % file_name)
+            json_schema_file = open('schemata/%s' % file_name)
+            assert check_schema(json.load(json_schema_file))
```

Comment:
File access should be done in a context manager so that it gets closed correctly
```python
with open('schemata/%s' % file_name) as json_schema_file:
  assert check_schema(json.load(json_schema_file))
```